web/src/app/users/SendTestDialog.tsx

```diff
@@ -57,34 +59,33 @@ export default function SendTestDialog(
   const [now] = useState(DateTime.utc())
   const status = data?.userContactMethod?.lastTestMessageState?.status ?? ''
   const cmDestValue = data?.userContactMethod?.formattedValue ?? ''
-  const cmType: ContactMethodType = data?.userContactMethod?.type ?? ''
-  const lastTestVerifyAt = data?.userContactMethod?.lastTestVerifyAt ?? ''
-  const timeSinceLastVerified = now.diff(DateTime.fromISO(lastTestVerifyAt))
+  const cmType: ContactMethodType | '' = data?.userContactMethod.type ?? ''
+  const lastSent = data?.userContactMethod?.lastTestVerifyAt
+    ? DateTime.fromISO(data.userContactMethod.lastTestVerifyAt)
+    : now.plus({ day: -1 })
   const fromValue =
     data?.userContactMethod?.lastTestMessageState?.formattedSrcValue ?? ''
   const errorMessage = (error?.message || sendTestStatus.error?.message) ?? ''
 
+  const hasSent =
+    Boolean(data?.userContactMethod.lastTestMessageState) &&
```

Comment:
```suggestion
    Boolean(data?.userContactMethod?.lastTestMessageState) &&
```
Same as above

Reply:
If `userContactMethod` isn't present on `data`, wouldn't this throw an error? because it's trying to access `lastTestMessageState` of `undefined`? Or are you saying that `userContactMethod` will always be present in this scenario

Reply:
I believe it tries them in order and once it hits undefined on a ? it returns the wanted result, or undefined if none given after the ??.

My IDE isn't throwing any errors or warning regarding this code

Reply:
This code reads to me that `userContactMethod` may not always be present on `data`, but if it is, `lastTestMessageState` will be available

Reply:
Not getting the error on my IDE either. Not sure why it's different from the TypeScript playground, but I dismissed my change request